src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -1218,10 +1220,10 @@ public void checkMaxQGreaterThanMinQ(
     }
 
     public void checkMaxReactivePowerGreaterThanMinReactivePower(MinMaxReactiveLimits minMaxReactiveLimits, AttributeModification<Double> minimumReactivePowerInfo, AttributeModification<Double> maximumReactivePowerInfo, NetworkModificationException.Type exceptionType, String errorMessage) {
-        Double previousMinimumReactivePower = minMaxReactiveLimits.getMinQ();
-        Double previousMaximumReactivePower = minMaxReactiveLimits.getMaxQ();
-        Double minReactivePower = minimumReactivePowerInfo != null ? minimumReactivePowerInfo.getValue() : previousMinimumReactivePower;
-        Double maxReactivePower = maximumReactivePowerInfo != null ? maximumReactivePowerInfo.getValue() : previousMaximumReactivePower;
+        double previousMinimumReactivePower = minMaxReactiveLimits.getMinQ();
```

Comment:
Why this change ?

Reply:
it is highlighted by sonarLint to change the type to primitive type